pkg/model/credential.go

```diff
@@ -0,0 +1,13 @@
+package model
+
+type Credential struct {
+	ID             string   `json:"id,omitempty" yaml:"id,omitempty"`
+	Name           string   `json:"name,omitempty" yaml:"name,omitempty"`
+	ClientId       string   `json:"clientId,omitempty" yaml:"clientId,omitempty"`
```

Comment:
```suggestion
	ClientID       string   `json:"clientId,omitempty" yaml:"clientId,omitempty"`
```

Reply:
Why was this resolved it should be `ID` and not `Id`

Reply:
Why was this resolved it should be ID and not Id